src/app/components/keyboard/keyboard.component.ts

```diff
@@ -8,17 +8,19 @@ import { Component, EventEmitter, Input, Output } from '@angular/core';
 export class KeyboardComponent {
   @Output() pressedKeyValue = new EventEmitter<string>();
   @Input() disableKeys = false;
+  @Input() roundFound = false;
 
   firstKeyBoardRow = ['Q', 'W', 'E', 'R', 'T', 'Y', 'U', 'I', 'O', 'P'];
   secondKeyBoardRow = ['A', 'S', 'D', 'F', 'G', 'H', 'J', 'K', 'L', 'Ñ'];
-  thirdKeyBoardRow = ['⌫', 'Z', 'X', 'C', 'V', 'B', 'N', 'M', '➜'];
+  thirdKeyBoardRow = ['⌫', 'Z', 'X', 'C', 'V', 'B', 'N', 'M'];
+  sendKey = '➜';
 
-  disableKey(key: string): boolean {
-    return key === '➜' && !this.disableKeys;
+  disableSendKey(): boolean {
+    return !this.disableKeys;
```

Comment:
It's not necessary to have a function that check a variable in the controller. In the HTML u just have to put the variable to check if its true/false